dbt/adapters/athena/connections.py

```diff
@@ -45,6 +45,8 @@ class AthenaCredentials(Credentials):
     poll_interval: float = 1.0
     _ALIASES = {"catalog": "database"}
     num_retries: Optional[int] = 5
+    s3_data_dir: Optional[str] = None
+    s3_data_naming: Optional[str] = "schema_table"
```

Comment:
Do you consider this a breaking change, going from `uuid` (default) to `schema_table` as a default ? I wonder if we should stick to `uuid`. 

If I drop a table in Athena, and redeploy dbt, it will fail with:

```
HIVE_PATH_ALREADY_EXISTS: Target directory for table 'dev_jessedobbelaere_silver.users' already exists: s3://stag-athena-results-jesse-test/dbt/dbt-dev/dbt-jessedobbelaere/tables/dev_jessedobbelaere_silver/users. You may need to manually clean the data at location 's3://stag-athena-results-jesse-test/dbt/dbt-dev/dbt-jessedobbelaere/tables/921180dc-e576-4d5d-b1dd-3fadfd5c45de' before retrying. Athena will not delete data in your account.
```

Because the s3 data is only removed by dbt-athena automatically looking up the **existing** table's data location path. If the table does not exist, but the s3 folder of that table is not fully empty, the error happens. It might cause confusion for some users?

Reply:
I will change to default uuid

Reply:
@jessedobbelaere how about using by default schema_table_unique? It does give te possibility to keep a nicer structure in the bucket, but at the same time give the possibility to have a full unique location

Reply:
Yeah that sounds like best of both worlds 👍 Great!